integration-tests/testkit/schema-policy.ts

```diff
@@ -0,0 +1,191 @@
+import { RuleInstanceSeverityLevel, SchemaPolicyInput } from '@app/gql/graphql';
+import { graphql } from './gql';
+
+export const TargetCalculatedPolicy = graphql(`
+  query TargetCalculatedPolicy($selector: TargetSelectorInput!) {
+    target(selector: $selector) {
+      id
+      schemaPolicy {
+        mergedRules {
+          ...SchemaPolicyRuleInstanceFields
+        }
+        projectPolicy {
+          id
+          rules {
+            ...SchemaPolicyRuleInstanceFields
+          }
+        }
+        organizationPolicy {
+          id
+          allowOverrides
+          rules {
+            ...SchemaPolicyRuleInstanceFields
+          }
+        }
+      }
+    }
+  }
+
+  fragment SchemaPolicyRuleInstanceFields on SchemaPolicyRuleInstance {
+    rule {
+      id
+    }
+    severity
+    configuration
+  }
+`);
+
+export const OrganizationAndProjectsWithSchemaPolicy = graphql(`
+  query OrganizationAndProjectsWithSchemaPolicy($organization: ID!) {
+    organization(selector: { organization: $organization }) {
+      organization {
+        id
+        schemaPolicy {
+          id
+        }
+        projects {
+          nodes {
+            id
+            schemaPolicy {
+              id
+            }
+          }
+        }
+      }
+    }
+  }
+`);
+
+export const UpdateSchemaPolicyForOrganization = graphql(`
+  mutation UpdateSchemaPolicyForOrganization(
+    $selector: OrganizationSelectorInput!
+    $policy: SchemaPolicyInput!
+    $allowOverrides: Boolean!
+  ) {
+    updateSchemaPolicyForOrganization(
+      selector: $selector
+      policy: $policy
+      allowOverrides: $allowOverrides
+    ) {
+      error {
+        message
+      }
+      ok {
+        organization {
+          id
+          schemaPolicy {
+            id
+          }
+        }
+        updatedPolicy {
+          id
+          allowOverrides
+          updatedAt
+          rules {
+            rule {
+              id
+            }
+            severity
+            configuration
+          }
+        }
+      }
+    }
+  }
+`);
+
+export const UpdateSchemaPolicyForProject = graphql(`
+  mutation UpdateSchemaPolicyForProject(
+    $selector: ProjectSelectorInput!
+    $policy: SchemaPolicyInput!
+  ) {
+    updateSchemaPolicyForProject(selector: $selector, policy: $policy) {
+      error {
+        message
+      }
+      ok {
+        project {
+          id
+          schemaPolicy {
+            id
+          }
+        }
+        updatedPolicy {
+          id
+          updatedAt
+          rules {
+            rule {
+              id
+            }
+            severity
+            configuration
+          }
+        }
+      }
+    }
+  }
+`);
+
+export const INVALID_RULE_POLICY = {
+  rules: [
+    {
+      ruleId: 'require-kamil-to-merge-my-prs',
```

Comment:
in case you wonder if I will carefully review your PR, yes

Reply:
`cmd+f` -> `kamil`